docs/configuration.md

```diff
@@ -1215,6 +1215,13 @@ Apart from these, the following properties are also available, and may be useful
     if it is too small, <code>BlockManager</code> might take a performance hit.
   </td>
 </tr>
+<tr>
+  <td><code>spark.broadcast.checksum</code></td>
+  <td>true</td>
+  <td>
+    Controls whether to enable checksum for broadcast.
```

Comment:
If this is going to be documented, it should be documented in a way that's useful for end users. This typically means describing what turning it on and off does from an end-user perspective. Does it make things faster, slower? more reliable, compatible, etc? This description here pretty much reiterates what's in the property key itself.